multicluster/controllers/multicluster/core/remote_cluster_manager_test.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"github.com/golang/mock/gomock"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog/v2"
+	k8smcsapi "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+	"antrea.io/antrea/multicluster/controllers/multicluster/common"
+
+	mcsv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+)
+
+var (
+	rcmtScheme = runtime.NewScheme()
```

Comment:
this is unused. you can check this through execute `make golangci` in antrea root directory.

Reply:
Sure

Reply:
It is used in the last 3 lines actually